### PR TITLE
Support --config-name for recorder start command

### PR DESCRIPTION
**Pull request by Hackerman342**

### CODE REVIEW

**Comment by ethanrublee** [comment]
Please add a test for this.

**Review by ethanrublee — CHANGES_REQUESTED** [review]
Please add a test case for this.

File: py/farm_ng/core/event_service_recorder.py [hunk 1]
```diff
@@ -357,9 +368,13 @@ async def job():
 def client_start_command(_args):
     config_list, service_config = load_service_config(args)
     loop = asyncio.get_event_loop()
+    config_name: str = args.config_name
 
     async def job():
-        reply = await EventClient(service_config).request_reply("start", config_list)
+        reply = await EventClient(service_config).request_reply(
+            f"start/{config_name}",
```
Comment (ethanrublee) [inline py/farm_ng/core/event_service_recorder.py:375]: Yes makes sense!

Comment (Hackerman342) [reply py/farm_ng/core/event_service_recorder.py:375]: :sweat_smile:

**Review by guilhermedemouraa — APPROVED** [review]
Beautiful!

**Comment by ethanrublee** [comment]
For test, you add a simple change here to cover the new request/response api: py/tests/_asyncio/test_event_service_recorder.py

**Comment by Hackerman342** [comment]
@ethanrublee I've been working on this. Maybe I'm missing something, but I don't see it as a simple tweak to the existing tests. There is no test infrastructure for the `RecorderService`, only the `EventServiceRecorder`.

**Comment by ethanrublee** [comment]
Ok bummer.  I see that now.

**Comment by ethanrublee** [comment]
We could follow up with a test I guess then.

**Review by ethanrublee — APPROVED** [review]

**Pull request merged**